src/style/app.module.css

```diff
@@ -1435,12 +1491,12 @@ hr {
 }
 
 .customcell {
-  background-color: var(--light-blue) !important;
-  color: var(--bs-white) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`